smssync/src/main/java/org/addhen/smssync/presentation/util/Utility.java

```diff
@@ -278,6 +278,9 @@ public static String getPhoneNumber(@NonNull Context context, @NonNull PrefsFact
      * @return String
      */
     public static String capitalizeFirstLetter(String text) {
+        if(text == null) return null;
```

Comment:
Can't find the relevant bit of that document; would the following be appropriate?
```java
f(text == null) {
	return null;
} else if(text.length() == 0) {
	return "";
} else if(text.length() == 1) {
	return text.toUpperCase();
} else {
	return text.substring(0, 1).toUpperCase() + text.substring(1);
}
```